server/src/main/java/com/distkv/server/metaserver/client/DmetaClient.java

```diff
@@ -63,6 +64,24 @@ public HeartbeatResponse heartbeat(NodeInfo nodeInfo) {
     }
   }
 
+  public GetGlobalViewResponse getGlobalView() {
+    // Get leader.
+    final PeerId leader = RouteTable.getInstance().selectLeader(RAFT_GROUP_ID);
+    final GetGlobalViewRequest request = new GetGlobalViewRequest();
+
+    try {
+      GetGlobalViewResponse response = (GetGlobalViewResponse) cliClientService.getRpcClient()
+          .invokeSync(leader.getEndpoint().toString(), request, HEARTBEAT_TIMEOUT);
```

Comment:
Why is HEARTBEAT_TIMEOUT needed here?

Reply:
We need a way to make dmeta and storeserver have the same heartbeat time

Reply:
It's better to set it as a static final variable for both meta server and store server.